docs/guide/release-notes/1.2.0.md

```diff
@@ -0,0 +1,75 @@
+---
+title: lit-html 1.2.0
+slug: lit-html-1-2-0
+tag: release-notes
+---
+
+See the [changelog](https://github.com/Polymer/lit-html/blob/master/CHANGELOG.md) for a complete list of changes.
+
+## `live` directive
+
+[Documentation](../template-reference#live)
+
+The `live` directive solves the problem of DOM state changing from underneath lit-html, for example with an `<input>` that a user can type into:
+
+```js
+html`<input .value=${live(x)}>`
+```
+
+In some cases you will want `x` to overwrite what the user has typed in. lit-html tracks the previous value of a binding and only updates the DOM is the value has changed. Without the live directive, lit-html might not think that the `value` property of the input has changed and won't update the DOM.
+
+The live directive ignores the previous value of the binding and always checks the current value in the DOM, causing the binding to update the input.
+
+We still reccomend that you handle user input so that the state and the DOM agree:
+
+```ts
+let text = '';
+const onInput = (e) => {
+  text = e.target.value;
+  go();
+};
+
+const go = () => {
+  html`<input .value=${text} @input=${onInput}>`
+}
+go();
+```
+
+## `templateContent` directive
+
+[Documentation](../template-reference#templatecontent)
+
+The `templateContent` directive lets you stamp out HTML templates into lit-html templates. This is useful in a number of cases where HTML `<template>` elements are provided by users of elements or parts of a build system.
+
+HTML:
+```html
+<template id="example">
+  <p>HTML Template</p>
+</template>
+```
+
+JavaScript:
+```js
+const template = document.querySelector('#example');
+html`
+  <h1>Example</h1>
+  ${templateContent(template)}
+`;
+```
+
+## `unsafeSVG` directive
+
+[Documentation](../template-reference#unsafesvg)
+
+`unsaveSVG` is the missing partner of [`unsafeHTML`](../template-reference#unsafehtml). It lets you render a frangment of SVG text as SVG elements rather than text. As with `unsafeHTML` this directive not safe to use with user-provided input, since if the input has `<script>` tags in it, there may be ways to get them to execute, etc.
+
+`unsafeSVG` creates elements in the SVG namespace, so it's for use inside of `<svg>` tags or inside of lit-html ``` svg`` ``` templates. If the input contains an `<svg>` tag itself, continute to use `unsafeHTML`.
```

Comment:
We haven't used this format (``` html`` ```) in discussing HTML templates, and it looks a little strange here. 

Maybe just use `svg` and link it to the API doc page so it's clear what we're talking about? 
``` [`svg`](/api/modules/lit_html.html#svg) ```

Reply:
Done, but that link destination leaves a lot to be desired. It doesn't describe what `svg` does at all, and I just noticed we don't mention it anywhere in the docs. I'll open a new issue for add it to "Writing Templates".

Reply:
👍